### DISCUSSION TITLE
Increase memory allowance?

### DISCUSSION THREAD

**Discussion by ssardina** [idrave/FOND-ASP#25] [Q&A] [answered]
Hola @idrave 

Como subo el limite de memoria para el planner? 

[Aca](https://github.com/idrave/FOND-ASP/issues/21#issuecomment-913617012) decis que te dio SAT, espectacular!  A mi me da memory out, que tengo que usar?

**Comment by idrave** [comment, marked as answer]
Para cambiar el límite es con la opción `-memout`, seguido del nuevo límite de memoria en bytes.
